modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-impl/src/main/java/com/liferay/headless/commerce/admin/order/internal/resource/v1_0/OrderItemResourceImpl.java

```diff
@@ -293,7 +293,10 @@ public OrderItem putOrderItem(Long id, OrderItem orderItem)
 
 		CommerceOrderItem commerceOrderItem =
 			_commerceOrderItemService.updateCommerceOrderItem(
-				null, id, GetterUtil.getString(orderItem.getOptions(), "[]"),
+				GetterUtil.getString(
+					orderItem.getExternalReferenceCode(),
+					commerceOrder.getExternalReferenceCode()),
+				id, GetterUtil.getString(orderItem.getOptions(), "[]"),
```

Comment:
We should not touch this method since it will most likely be deprecated soon. I will inform users to use byERC instead.